doc/userguide/rules/smb-keywords.rst

```diff
@@ -0,0 +1,245 @@
+SMB Keywords
+==============
+
+SMB keywords used in both SMB1 and SMB2 protocols.
+
+smb.cmd
+---------
+
+Used to match SMB command. Decimal, Hexadecimal or command names (case
+insensitive) values are admitted.
+
+
+The following signature are equivalent::
+
+  alert smb any any -> any any (msg: "Smb command rule"; smb.cmd: 10; sid: 1;)
+  alert smb any any -> any any (msg: "Smb command rule"; smb.cmd: 0xa; sid: 1;)
+  alert smb any any -> any any (msg: "Smb command rule"; smb.cmd: lock; sid: 1;)
+
+
+You can also specify several commands separated by comma::
+
+  alert smb any any -> any any (msg: "Smb command rule"; smb.cmd: 10,create,0x4; sid: 1;)
```

Comment:
what is the logic here? A command can only be one thing, so it matches as if `10 or create or 0x4`?